frontend/public/index.html

```diff
@@ -2,8 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
I see the previous icon was removed. Let's keep it for now until we get new designs.

Reply:
done